drivers/gpio/gpio_pcal64xxa.c

```diff
@@ -119,6 +125,9 @@ struct pcal64xxa_drv_cfg {
 	const struct gpio_dt_spec gpio_reset;
 	const struct gpio_dt_spec gpio_interrupt;
 	const struct pcal64xxa_chip_api *chip_api;
+#ifdef CONFIG_GPIO_PCAL64XXA_RETENTION
```

Comment:
See `DT_ANY_INST_HAS_PROP_STATUS_OKAY` to make inclusion automatic

Reply:
done